include/image.h

```diff
@@ -42,6 +42,11 @@
 #endif
 #endif
 
+#define likely(x)       __builtin_expect((x),1)
```

Comment:
You might need to wrap these in `__GNUC__`?

Reply:
Code has been made gcc-arm only